File: Dart/1015.dart
import 'dart:io';
import 'dart:math';

void main() {
  String input1 = stdin.readLineSync();
  String input2 = stdin.readLineSync();
  List x, y;
  x = input1.split(" ");
  y = input2.split(" ");
  x = x.map((e) => double.parse(e)).toList();
  y = y.map((e) => double.parse(e)).toList();
  print(
      sqrt((pow((y[1] - x[1]), 2) + pow((x[0] - y[0]), 2))).toStringAsFixed(4));
}

File: JavaScript/1015.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

var x = lines.shift();
var y = lines.shift();

var x1 = parseFloat(x.split(" ")[0]);
var y1 = parseFloat(x.split(" ")[1]);

var x2 = parseFloat(y.split(" ")[0]);
var y2 = parseFloat(y.split(" ")[1]);

console.log(
  Math.sqrt((x1 - x2) * (x1 - x2) + (y1 - y2) * (y1 - y2)).toFixed(4)
);
